debugprofiler.md

```diff
@@ -0,0 +1,52 @@
+Forge Debug Profiler oraz jego interpretacja
+===============================================
+
+Spis treści
+-----------
+* [Podstawy](#podstawy)
+* [Jak używać debugera](#Jak_używać_debugera)
+* [Czytanie Profilera](#czytanie)
+
+## Podstawy
+Minecraft dostarcza nam `Debug Profiler` dzięki któremu można sprawdzić obciążający kod.
+Szczególnie chodzi tutaj o rzeczy pokroju `TickEvent` lub `Ticking TileEntities`, 
```

Comment:
Lepiej było by napisać, że cały uśrednioty "tick" rozumiany jako jedno przejście pętli głównej gry jest rozłożony na czynniki pierwsze, a przy każdym z tych elementów dostajemy procent czasu z całej petli, jaki przypada na ich wykonanie.

Reply:
Imo zbyt skomplikowane, nie można prościej? Mimo iż mówimy tutaj o trochę zaawansowanych rzeczach, przeznaczone jest to dla laika.

Reply:
Zostawiam do oceny arturowi i bopkowi.

Reply:
Biorąc pod uwagę jak wiele osób na discordzie craftserve ma problem ze znacznie prostszymi rzeczami, myślę, że nie ma co w ogóle z tym materiałem celować do takich kompletnie zielonych osób. Moim zdaniem jest ok.

Reply:
Ok, w następnym commicie poprawie